build/yamls/base/conf/antrea-agent.conf

```diff
@@ -147,3 +151,7 @@ featureGates:
 
 # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
 #tlsMinVersion:
+
+# A string slice of values which specifies the host IPv4/IPv6 addresses for NodePort. Values can be valid IP blocks.
+# (e.g. 1.2.3.0/24, 1.2.3.4/32). An empty string slice is meant to select all host IPv4/IPv6 addresses.
```

Comment:
s/slice/array - slice is a Go concept and this is a YAML file. Although based on other parameters, maybe this should just be a string (comma-separated list)?

Reply:
String can also work, then we need to split the string. The results are the same.